registry/storage/blobstore.go

```diff
@@ -141,6 +141,7 @@ func (bs *blobStore) resolve(ctx context.Context, path string) (string, error) {
 type blobStatter struct {
 	driver driver.StorageDriver
 	pm     *pathMapper
+	tomb   *tomb
```

Comment:
Good catch  That was an artefact from a previous version - It was even being referenced.